bin/elasticsearch.bat

```diff
@@ -1,10 +1,46 @@
 @echo off
 
-SETLOCAL
+SETLOCAL enabledelayedexpansion
 TITLE Elasticsearch ${project.version}
 
+SET params='%*'
+
+:loop
+FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
+    SET current=%%A
+    SET params='%%B'
+	SET silent=N
+	
+	IF "!current!" == "-s" (
+		SET silent=Y
+	)
+	IF "!current!" == "--silent" (
+		SET silent=Y
+	)	
+	
+	IF "!silent!" == "Y" (
+		SET nopauseonerror=Y
+	) ELSE (
```

Comment:
But this is working:

```
$ java -jar injector.jar -silent
```

As long as - silent is at the end. Would this work with:

```
"%JAVA_HOME%\bin\java" %JAVA_OPTS% %ES_JAVA_OPTS% -cp "%ES_CLASSPATH%" "org.elasticsearch.bootstrap.Elasticsearch" %ES_PARAMS% 
```